src/api/contributionsApi.test.ts

```diff
@@ -36,4 +42,29 @@ describe('fetchDefaultEpic', () => {
             highlighted: ['Highlighted Text'],
         });
     });
+
+    it('caches successful epic fetches', async () => {
+        fetchMock.get(epicUrl, epicResponse);
+
+        await fetchDefaultEpicContent();
+        await fetchDefaultEpicContent();
+
+        expect(fetchMock.calls().length).toEqual(1);
+    });
+
+    it('does not cache unsuccessful epic fetches', async () => {
+        fetchMock.get(epicUrl, { status: 500 });
+
+        try {
+            await fetchDefaultEpicContent();
+        } catch {}
+
+        expect(fetchMock.calls().length).toEqual(1);
```

Comment:
Is this *too* thorough?

Reply:
Ha yeah maybe. I didn't have this to begin with but I thought it might help in understanding what was going on in the test. Maybe it's overkill though. I'll remove for now.